src/common/HaskellDo/Toolbar/State.hs

```diff
@@ -30,8 +33,14 @@ initialState = State
     , projectConfig = ""
     , projectOpened = False
     , createProject = False
+    , directoryExists = False
+    , directoryList = ([], [])
     }
 
+
+lastProjectFile :: FilePath
+lastProjectFile = "lastproject"
```

Comment:
Maybe we could even have a config loader and store stuff like `lastProjectFile`, `appPort`, etc... in a config file like a `TSON` (Typed JSON)? Anyways, this should be in another PR